shared/base/view.js

```diff
@@ -398,6 +398,10 @@ module.exports = BaseView = Backbone.View.extend({
   },
 
   remove: function() {
+    // Remove reference to this view from its parentView
+    var index = _.indexOf(this.parentView.childViews, this);
+    this.parentView.childViews.splice(index, 1);
```

Comment:
This might be easier to just write as `this.parentView.childViews = _.without(this.parentView.childViews, this)`